### ISSUE TITLE
Be notified only of PRs where I'm assigned as reviewer

### ISSUE THREAD

**Issue by JordanGottardoDocplanner** [integrations/slack#1606]
**Is your feature request related to a problem? Please describe.**
I don't see a clear way of receiving notifications for only the PRs I am assigned to as a reviewer. 
I subscribed to a repo using the command `/github subscribe <organization>/<repository> pulls `.
This will notify me whenever a PR is opened. However, I'd like to be notified only when I'm a reviewer of that PR. Since the repository is pretty big, I am being notified for PRs not concerning my work and it's very noisy.

**Describe the solution you'd like**
Be notified only of PRs where I'm a reviewer

**Comment by MM3y3r** [comment]
Has this been solved?

**Comment by Lp-Francois** [comment]
Hello 👋
Still not updates? 😅
Especially for monorepos, seems like a standard expected feature

**Comment by henryhobhouse** [comment]
Also +1 on this for monorepos. Otherwise its a noisy mess and defeats the object.

**Comment by huseyin-hamilton-cko** [comment]
+1

**Comment by neighborcaroline** [comment]
@JordanGottardoDocplanner (not sure how to notify everyone on this post but) this can be done via Scheduled Reminders! Check out our bud Rasim's [blog post](https://rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up/) to learn how 🎉

**Comment by andrii-motorway** [comment]
https://rasim.pro/blog/personal-slack-github-notifications-for-pull-requests-reviewers-how-to-set-up/

**Comment by robertojrdev** [comment]
+1 also for mentions - I cannot use the Scheduled Reminders @neighborcaroline and @andrii-motorway mentioned because the organization doesn't want to set it up.

**Comment by senghorn** [comment]
+ 1 really need this

**Comment by ish-joshi** [comment]
This would be super helpful! A large repo results in unnecessary notifications based on repo name only.

**Comment by setvik** [comment]
Would be really valuable

**Comment by eric-shiu** [comment]
This would be super helpful for me

**Comment by oriheimlich** [comment]
Will be happy for that as well

**Comment by nmagedman** [comment]
**TL;DR "Scheduled Reminders" provides the solution you want**

A solution was provided above by @neighborcaroline (and @andrii-motorway) involving "Scheduled Reminders".  I initially dismissed that solution as being off-the-mark, since I wasn't looking for daily reminders -- I wanted _instant_ notifications of new events.

However, it ought to be clarified that this solution _really will_ enable immediate notifications, not just daily scheduled reminders.  **The feature is misnamed**.  Or rather, _misconceptualized_.  The GitHub/Slack integration ought to be framed as a _connector_ enabling GitHub to send messages to you via Slack.  Those messages can be:

  - "A PR was just assigned to you"
  - "Someone commented on your PR"
  - Any of several other events
  - A daily summary of still-open PRs assigned to you
  
The developers at GitHub, however, conceptualized (and named) the integration specifically as a "Scheduled Reminders" feature, which -- oh, also _happens_ to be able to send "real-time" alerts.

I can't emphasize explicitly and strongly enough that GitHub’s _naming_ the integration "Scheduled Reminders" really masks its true value as an event notification system.  The daily reminder aspect of it ought to be downplayed as a nice extra feature, not given the center-stage spotlight.

So... follow the instructions in blog post.  Add your Slack Workspace.  Enable the real-time alerts.  And, personally, I've disabled the daily Scheduled Reminders by unchecking all 7 days from the list, since (a) it drowned out the real-time alert message history in Slack and (b) the real-time alerts auto-update and ~strikethrough~ themselves when the event is no longer relevant.

One last note: When adding the workspace, it shows (at least to me) a scary popup dialog re: migrating from a legacy app to the new one by uninstalling and re-installing, and it had a link to documentation that no longer exists.  I was very worried about breaking things for the rest of the company.  But it was fine -- much ado about nothing -- go ahead and add the workspace.

**Comment by victorlee134** [comment]
+1

**Comment by alankritjoshi** [comment]
Why is this still an unsolved issue when github and slack are both leaders in their respective product categories? 

The primary point of the integration should be to allow developers to quickly notify other reviewers and act on notifications that require their review.

I always end up installing other apps like toast, pullnotifier, etc. (which are all paid at enterprise level btw) to achieve this. The slackbot right now is all unactionable noise.

**Comment by zwang1-godaddy** [comment]
+1

**Comment by eangermann-r7** [comment]
+1

**Comment by gabriel-pineda** [comment]
If anyone is looking for an alternative, toast and [pullnotifier](https://pullnotifier.com/) are great options. We've been using it to simplify our workflow, and it's been a game-changer for us.

**Comment by kaleb-cedar** [comment]
I'm pretty sure this issue should be closed based on [this comment](https://github.com/integrations/slack/issues/1606#issuecomment-2179076293).

Real-time alerts exist for the GitHub Slack integration: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/managing-your-scheduled-reminders#:~:text=Optionally%2C%20to%20receive%20Slack%20messages%20as%20soon%20as%20certain%20events%20happen%2C%20select%20Enable%20real%2Dtime%20alerts%20and%20select%20the%20events%20to%20receive%20real%2Dtime%20alerts%20for.

**Comment by martimlobao** [comment]
This seems like something that should be doable through the Github app in Slack. I'm satisfied that the solution described above does exactly what I wanted, but its lack of discoverability through Github's Slack app meant that I thought it simply wasn't possible. Adding a way to enable these notifications from Slack is all that remains to be done for this ticket to be closed (honestly, even just sending a user a guide on how to enable these settings is probably fine imo).

**Comment by dandu1008** [comment]
+1
